common/service/dynamicconfig/constants.go

```diff
@@ -703,6 +708,13 @@ func DomainFilter(name string) FilterOption {
 	}
 }
 
+// DomainIDFilter filters by domain id
+func DomainIDFilter(name string) FilterOption {
```

Comment:
nit: probably rename the parameter as `domainID` instead of `name`?

Reply:
done